introduction.md

```diff
@@ -1,3 +1,10 @@
 Hi, my name is Geanderson and I'm Brazilian. I'm passionate for technology since
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?